avocado/plugins/wrapper.py

```diff
@@ -18,7 +18,7 @@
 from avocado.core import exit_codes
 from avocado.core.output import LOG_UI
 from avocado.core.plugin_interfaces import CLI
-from avocado.utils import process
+from avocado.utils import astring, process
```

Comment:
Even though inspekt won't catch it, this is pretty much against our style of imports.  It'd be more consistent to have each one on a separate line.

Reply:
sure